antora-ui-camel/src/css/doc.css

```diff
@@ -577,10 +596,6 @@
   display: none;
 }
 
-b.button {
-  white-space: nowrap;
-}
-
```

Comment:
I can't find any HTML markup that uses this, can you help me, what is this change about?

Reply:
I observed that in the mobile view the navbar hamburger moves to the side because of the width occupied by the paragraph and links. Hence, removing the whitespace and word-wrap ensures continuous flow of the page and also the page becomes responsive.

Reply:
If you observe in pages such as the blog page, the mobile view isn't responsive and the navbar hamburger isn't visible. Similar cases for other pages and the changes resolve that.

Reply:
Yup seeing it now.